package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "version": "1.0.0",
+  "description": "My webpack project",
+  "name": "my-webpack-project",
+  "scripts": {
+    "build": "webpack --mode=production --node-env=production",
+    "build:dev": "webpack --mode=development",
+    "build:prod": "webpack --mode=production --node-env=production",
+    "watch": "webpack --watch",
+    "serve": "webpack serve"
+  },
```

Comment:
- [ ] Something important with webpack, is run the project for production, development, test, etc. I kindly invite to use this here: 🐶 
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack serve --open",
    "watch": "webpack --watch --mode development"
};
```